### PR TITLE
Add the ability to set preferences from the tray-icon

### PR DESCRIPTION
**Pull request by csm10495**
I wanted the ability to quickly snooze without having to open the GUI and see my messages. It's a nice feature when someone comes to your desk and you'd like popups to not come up. :) 

Anyways, this pull request duplicates the preferences menu since I figure more than snooze may be useful to be able to change without opening the whole interface.

### CODE REVIEW

File: resources/js/menu.js [hunk 1]
```diff
@@ -13,65 +13,70 @@
...
+        },
+        { label: "3 hours", type: 'checkbox', checked: preferences.isSnoozeActive() && preferences.currentSnoozeSelection() == "3_hours", click() {
+          preferences.snooze("3_hours") }
+        },
+        { label: "12 hours", type: 'checkbox', checked: preferences.isSnoozeActive() && preferences.currentSnoozeSelection() == "12_hours", click() {
+          preferences.snooze("12_hours") }
+        }
+      ] }
+    ]
+  }, { type: 'separator' }, {
```
Comment (klinker24) [inline resources/js/menu.js]: I would remove this preference section and put it down below, in the `buildMenu` function. Let's change the preferences in the menu bar to just be the "Notification Preferences".

**Review by klinker24 — COMMENTED** [review]
I would say, if we are going to do this, let's only make the notification preferences available in the tray. I don't think we need the ability to "Show in Menu Bar/Tray" in this menu, and you actually missed moving the "Show Unread Count on Icon" option as well. 

I like the idea of doing this though, and will be more than happy to merge it, with this change and after I test it a bit more. Thanks for the PR!

**Comment by csm10495** [comment]
@klinker24, I've updated the PR to fix the things you mentioned. Please take another look. Thanks!

**Comment by klinker24** [comment]
Sorry, little misunderstanding on the requested changes.

I didn't actually want you to move the "Show Unread Count On Icon" preference to the menu bar menu, I wanted that one to stay in the main preferences page, where it was at. I just was looking to move the "Show in Menu Bar" option back out to the main preferences tab, instead of the menu bar's menu, where you had put it in the PR.

All we really need to do here is move that "Show Unread Count on Icon" preference back to where it was, then this can be done.

The main preferences menu will have `Notification Preferences`, `Show in Menu Bar`, and `Show Unread Count on Icon`, just like it currently has with `v2.1.0`. 

The menu bar's menu will just have the `Notification Preferences`

**Comment by csm10495** [comment]
@klinker24, one more try. Please take a look and let me know if I'm still off. Thanks.

**Comment by klinker24** [comment]
I went ahead and implemented it here: https://github.com/klinker24/messenger-desktop/commit/6f8dccd1cbac4c6f514f78da0f42f23dfdfb6227

You were close, but the menus just weren't really matching up to what is already in `v2.1.0`. I just extracted the notification preferences, using your code, and didn't touch any of the the other menus.

Thanks for the help on this

**Pull request closed without merging**